src/brsTypes/components/MarkupGrid.ts

```diff
@@ -0,0 +1,46 @@
+import { FieldModel } from "./RoSGNode";
+import { AAMember } from "./RoAssociativeArray";
+import { ArrayGrid } from "./ArrayGrid";
+
+export class MarkupGrid extends ArrayGrid {
+    readonly defaultFields: FieldModel[] = [
+        { name: "itemComponentName", type: "string", value: "" },
+        { name: "content", type: "node" },
+        { name: "itemSize", type: "array", value: "[0,0]" },
+        { name: "numColumns", type: "integer", value: "0" },
+        { name: "numRows", type: "integer", value: "12" },
+        { name: "rowHeights", type: "array", value: "[]" },
+        { name: "columnWidths", type: "array", value: "[]" },
+        { name: "rowSpacings", type: "array", value: "[]" },
+        { name: "columnSpacings", type: "array", value: "[]" },
+        { name: "fixedLayout", type: "boolean", value: "false" },
+        { name: "imageWellBitmapUri", type: "string", value: "" },
+        { name: "drawFocusFeedback", type: "boolean", value: "true" },
+        { name: "drawFocusFeedbackOnTop", type: "boolean", value: "false" },
+        { name: "focusBitmapUri", type: "string", value: "" },
+        { name: "focusFootprintBitmapUri", type: "string", value: "" },
+        { name: "focusBitmapBlendColor", type: "string", value: "0xFFFFFFFF" },
+        { name: "focusFootprintBlendColor", type: "string", value: "0xFFFFFFFF" },
+        { name: "wrapDividerBitmapUri", type: "string", value: "" },
+        { name: "wrapDividerHeight", type: "float", value: "0.0" },
+        { name: "sectionDividerBitmapUri", type: "string", value: "" },
+        { name: "sectionDividerFont", type: "node" },
+        { name: "sectionDividerTextColor", type: "string", value: "0xddddddff" },
```

Comment:
Tiny nit, but curious why hex casing is inconsistent in color values.

Reply:
Oh whoops, I just copy-pasted from the docs and didn't even notice, I can make them all uppercase.